src/components/playlist/playlistList.tsx

```
@@ -0,0 +1,117 @@
+    openSuccessToast,
+    handleNextVideo
+}: PlaylistListProps) => {
+
+    const { isAdmin, nickname, playlistCode } = useAuthCheck();
+    const { currentIndex, isDeleteMode } = usePlaylistContext();
+
+    if (!playlistCode) return null;
+
+    const { removeVideo } = usePlaylistActions(playlistCode);
+
+    const handleVideoDelete = (e: Event, item: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)